woocommerce/woocommerce
# Add React Testing Library to the Woo AI plugin.

### Changes proposed in this Pull Request:

Add React Testing Library to the Woo AI plugin.
Add a super simple test to verify that RTL is working in order to keep this PR small.

### How to test the changes in this Pull Request:

1. Run `pnpm install` if it doesn't automatically.
2. Run `pnpm run test:unit`.
3.  Verify that tests pass. :)
